app/build.gradle

```diff
@@ -71,6 +71,10 @@ tasks.dockerPrepare.configure {
     dependsOn(tasks.bootJar)
 }
 
+docker {
+    noCache false
+}
+
```

Comment:
Anyone see a problem with using this setting (allowing docker to use image caches and not forcing it to recreate the same image layer) for all projects with `Dockerfile`s? This change only applies to the `app` image.

Reply:
will it recreate the image if the Dockerfile changes?

Reply:
It should. You can double-check by running `./gradlew :app:docker` and it should say `CACHED` if nothing has changed:
```
#6 [3/7] WORKDIR /project
#6 sha256:60ed93b080ff8a664f6b2614f423ad674ce0b98e8e61f8b33c9fa9b3d1127f68
#6 CACHED

#8 [4/7] COPY entrypoint.sh entrypoint.sh
#8 sha256:0b74e2307f05f49125a38ef5ff143ebd80d5a38471494b144ac2332f3d70931e
#8 CACHED
```

Reply:
then I don't see a problem with that